### ISSUE TITLE
Using CSLA 5: Blazor and WebAssembly book online

### ISSUE THREAD

**Issue by rockfordlhotka** [MarimerLLC/cslaforum#887]
I am excited to announce that the "Using CSLA 5: Blazor and WebAssembly" book is now available for purchase!

https://store.lhotka.net/using-csla-5-blazor-and-webassembly

**Comment by JacoJordaan** [comment]
@rockfordlhotka great stuff.
Will you please check the shopping card. It is giving an internal error.

**Comment by rockfordlhotka** [comment]
Should be fixed now @JacoJordaan - thank you!

**Comment by itgoran** [comment]
Yes, it's fixed. I was able to make a purchase.

**Comment by Chicagoan2016** [comment]
Morning folks, started reading the book and can't view table of contents in pdf  :)  The draft versions of the books had them.
Regards

**Comment by Chicagoan2016** [comment]
so May is the month!!!! according to this article

https://visualstudiomagazine.com/articles/2020/01/28/blazor-webassembly-preview.aspx

**Comment by BlagoCuljak** [comment]
Hey @rockfordlhotka ,if I buy book today, will I be able to download updated edition when client side WA is finished?

**Comment by rockfordlhotka** [comment]
Yes @BlagoCuljak, my plan is to update the book when Blazor WebAssembly is released (in May?) and you'll be able to return to the store and download the updated edition.

**Comment by rockfordlhotka** [comment]
@Chicagoan2016 thanks for letting me know about the TOC issue - I'll look into that.
